projects/project-2/readings/Project 2 - SPARQL library

```diff
@@ -80,6 +96,10 @@ This situation presents a logical inconsistency within the ontology, as disjoint
 
 5. Identifying Classes with Different Labeling Standards
 
+Title: Inconsistent Class Labeling
+Constraint Description: Detects classes with different labeling standards.
+Severity: Warning
+
```

Comment:
This one gets all the classes which are different and have different labels. I am not sure that's what you were trying to achieve